Gemfile.lock

```diff
@@ -47,11 +49,12 @@ PLATFORMS
 DEPENDENCIES
   jekyll (= 3.4.3)
   jekyll-feed (~> 0.6)
+  jekyll-sitemap (~> 1.1)
   minima (~> 2.0)
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.4.0p0
+   ruby 2.1.9p490
```

Comment:
Can you try to stash those changes away? and just keep the jekyll* related changes and see if it still works?